lighthouse-core/formatters/partials/cards.html

```diff
@@ -0,0 +1,8 @@
+<ul class="subitem__details cards__container">
+  {{#each this}}
+    <div class="subitem__detail scorecard" {{#if snippet}}title="{{snippet}}"{{/if}}>
```

Comment:
I reserved `title` for the card title. Also wanted to make it obvious this was a code `snippet`.